python/mxnet/metric.py

```diff
@@ -917,6 +917,73 @@ def update(self, labels, preds):
             self.sum_metric += (-numpy.log(prob + self.eps)).sum()
             self.num_inst += label.shape[0]
 
+@register
+@alias('log_loss')
+class LogarithmicLoss(EvalMetric):
```

Comment:
where does the name come from?

Reply:
The log loss used in avazu kaggle competition 
https://www.kaggle.com/wiki/LogLoss